src/components/EntryForm.jsx

```diff
@@ -49,9 +49,8 @@ class Form extends Component {
                 <FieldElement className="pure-input-1" name="category" component="select">
                   <option value={-1}>- {t("chooseCategory")} -</option>
                   <option value={IDS.INITIATIVE}>{t("category." + NAMES[IDS.INITIATIVE])}</option>
-                  {/* this functionality will be made in future */}
-                  {/* <option value={IDS.COMPANY}>{t("category." + NAMES[IDS.COMPANY])}</option> */}
-                  <option value={IDS.EVENT}>{t("category." + NAMES[IDS.EVENT])}</option>
```

Comment:
@dmitrap why do you commit the `event` option?

Reply:
nevermind that

Reply:
thx for revert chagnes